### PR TITLE
Osmolbp CLI client bootstrapping.

### PR DESCRIPTION
**Pull request by dayakarreddyn**
## What is the purpose of the change

This PR bootstraps cli client functionality.

## Brief Changelog
 
  - Added CLI client 

## Testing and Verifying
This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? yes
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? no
  - How is the feature or change documented? not applicable

### CODE REVIEW

File: x/osmolbp/api/msg_create_lbp.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ func (msg *MsgCreateLBP) validate() []string {
 	}
 
 	return errmsgs
+	return errmsgs
```
Comment (robert-zaremba) [inline x/osmolbp/api/msg_create_lbp.go:52]:
```suggestion
```

File: x/osmolbp/keeper/query_server.go [hunk 2]
```diff
@@ -27,6 +27,31 @@ func (k Keeper) LBPs(goCtx context.Context, q *api.QueryLBPs) (*api.QueryLBPsRes
...
+	if err != nil {
+		return nil, err
+	}
+	poolId := storeIntIdKey(q.LbpId)
+	up, err := k.getUserPosition(store, poolId, user, false)
+	if err != nil {
+		return nil, err
+	}
+	return &api.QueryUserPositionResponse{up}, nil
+>>>>>>> 2de0aefed795f8f824a6066c00e2720f33e423bd
```
Comment (robert-zaremba) [inline x/osmolbp/keeper/query_server.go:54]: duplicate, the code above should be removed.

File: x/osmolbp/keeper/store.go [hunk 3]
```diff
@@ -126,4 +126,12 @@ func storeIntIdKey(id uint64) []byte {
...
 	return idBz
+<<<<<<< HEAD
 }
+=======
+<<<<<<< HEAD
+}
+=======
+}
+>>>>>>> 2de0aefed795f8f824a6066c00e2720f33e423bd
+>>>>>>> osmosis-labs-osmolbp
```
Comment (robert-zaremba) [inline x/osmolbp/keeper/store.go:137]: need to solve the conflict

File: x/osmolbp/client/cli/flags.go [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+	fs.String(FlagTokenIn, "", "denom used to buy LB tokens.")
+	fs.String(FlagTokenOut, "", "token denom to be bootstrapped.")
+	fs.String(FlagStartTime, "", "when the token sale starts.")
+	fs.String(FlagDuration, "", "time that the sale takes place over.")
+	fs.String(FlagInitialDeposit, "", "total number of `tokens_out` to be sold during the continuous sale.")
+	fs.String(FlagTreasury, "", "account which provides the tokens to sale and receives")
+
+	return fs
+}
```
Comment (robert-zaremba) [inline x/osmolbp/client/cli/flags.go:30]: migration to JSON based arguments will be done later?

**Review by robert-zaremba — APPROVED** [review]
pre-approving. Left few comments. There are conflicts to solve.

File: x/osmolbp/client/cli/flags.go (hunk 4 above)
Comment (dayakarreddyn) [reply x/osmolbp/client/cli/flags.go:30]: Yes

**Comment by ValarDragon** [comment]
Why did every proto file change? Was there some change to proto compilation tooling merged?

**Comment by ValarDragon** [comment]
Oh this is being merged to the LBP branch. Feel free to squash merge Robert

**Comment by robert-zaremba** [comment]
yes, at the end I will regenerate the files anyway

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**